Fix CAPI getting-started  tutorial (#643)
canonical/k8s-snap: docs/src/capi/tutorial/getting-started.md

### Configure clusterctl

`clusterctl` contains a list of default providers. Right now, {{product}} is 
not yet part of that list. To make `clusterctl` aware of the new

> providers, we need to add canonical providers to configuration

Reviewer: ```suggestion
providers we need to add add them to the configuration
```

### Configure clusterctl

`clusterctl` contains a list of default providers. Right now, {{product}} is 
not yet part of that list. To make `clusterctl` aware of the new
providers, we need to add canonical providers to configuration
file. Edit `~/.cluster-api/clusterctl.yaml` and add the following:

> 

Author: ```suggestion
```

before sourcing it. Then generate the cluster manifest:

```

> source ./templates/<infrastructure-provider>/cluster-template.rc

Author: ```suggestion
source ./templates/<infrastructure-provider>/template-variables.rc
```
